canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.scss

```diff
@@ -37,6 +37,29 @@ $operator-margin: 50%;
 }
 
 
+.properties-field-and-values-table-container, /* In Tearsheet Fields dropdown and table should occupy 50% width */
+.properties-expression-function-select, /* In Tearsheet Functions dropdown should occupy 50% width */
+.properties-functions-table-container { /* In Tearsheet Functions table should occupy 50% width */
+	width: 50%;
+	tbody {
+		tr {
+			border-bottom: 0;
```

Comment:
Why do we need this styling?

Reply:
This was related to table changes which I have reverted afterwards.